src/watchmaker/workers/salt.py

```diff
@@ -240,6 +241,19 @@ def _prepare_for_install(self):
         ) as fh_:
             yaml.safe_dump(self.salt_conf, fh_, default_flow_style=False)
 
+    def call_salt(self, command):
+        """Call a Salt command."""
+        raise NotImplementedError
+
+    def _check_salt_version(self):
+        current_salt_version = self.call_process(['salt-call', '--version'])
```

Comment:
Need to use a value for salt-call that works across both Linux and Windows...

```suggestion
        current_salt_version = self.call_process([self.salt_call, '--version'])
```